keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/config/ClouddriverConfiguration.kt

```diff
@@ -50,6 +51,7 @@ class ClouddriverConfiguration {
     CloudDriverService =
     Retrofit.Builder()
       .addConverterFactory(JacksonConverterFactory.create(objectMapper))
+      .addCallAdapterFactory(SpinnakerCallAdapterFactory)
```

Comment:
The "trick" is to add a custom call adapter to retrofit which adds the Spinnaker authz headers to the request.